airbyte-server/src/test/java/io/airbyte/server/handlers/OperationsHandlerTest.java

```diff
@@ -210,6 +213,7 @@ void testDeleteOperationsForConnection() throws JsonValidationException, IOExcep
     operationsHandler.deleteOperationsForConnection(sync, toDelete);
 
     verify(configRepository).writeStandardSyncOperation(operation.withTombstone(true));
+    verify(configRepository).updateConnectionOperationIds(syncConnectionId, Collections.emptySet());
```

Comment:
We should probably have a test case where this is not being called with the empty set

Reply:
Good call, updated test to cover both the empty case and the case where an operation remains